scripts/checker.py

```diff
@@ -0,0 +1,32 @@
+import os
+import imghdr
+from PIL import Image
+import shutil
+
+supported_image_ext = ('png', 'bmp', 'jpeg')
```

Comment:
@ken479 does jpeg cover both format of .jpeg and .jpg?

bmp is supported too
supported image extension of classifai is layout here
https://github.com/CertifaiAI/classifai/blob/v2_alpha/classifai-core/src/main/java/ai/classifai/util/type/FileFormat.java#L27-L33

for pdf, tif, tiff, not directly supported. these have to convert using the converter launcher
(so yeah , they fall under not supported category)

Reply:
@codenamewei ya, I think it supports both. Pillow can convert other image formats too.
[https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#](url)

Reply:
okay